### PR TITLE
[SW-537] convert TransactionList to composition api

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/components/TransactionList.vue [hunk 1]
```diff
@@ -78,135 +89,166 @@ export default {
...
+    const activeNetwork = useGetter('activeNetwork');
+    const getAccountPendingTransactions = useGetter('getAccountPendingTransactions');
+    const getTxSymbol = useGetter('getTxSymbol');
+
+    function compareCaseInsensitive(
+      str1: string,
+      str2: string,
+    ) {
+      return str1.toLocaleLowerCase() === str2.toLocaleLowerCase();
+    }
```
Comment (pstachel) [inline src/popup/components/TransactionList.vue:115]: As it's not strictly related to TransactionList maybe you could move it to utils?

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/TransactionList.vue [hunk 2]
```diff
@@ -78,135 +89,166 @@ export default {
...
+
+    const filteredTransactions = computed(() => {
+      const isFungibleTokenTx = (tr: any) => Object.keys(availableTokens.value)
```
Comment (pstachel) [inline src/popup/components/TransactionList.vue:118]: missing type

**Review by pstachel — CHANGES_REQUESTED** [review]

File: src/popup/components/TransactionList.vue (hunk 1 above)
Comment (konradodo) [reply src/popup/components/TransactionList.vue:115]: i do this on my other pr

File: src/popup/components/TransactionList.vue [hunk 3]
```diff
@@ -78,135 +89,166 @@ export default {
...
+      all: {}, in: {}, out: {}, dex: {},
     });
-    this.$watch(({ displayMode }) => displayMode, this.checkLoadMore);
-  },
-  methods: {
-    checkLoadMore() {
-      if (this.isDestroyed || !this.transactions.nextPageUrl) return;
+
+    const availableTokens = computed(() => root.$store.state.fungibleTokens.availableTokens);
+    const transactions = useState<any>('transactions');
```
[the remark this answers is not in the document]
Comment (konradodo) [reply src/popup/components/TransactionList.vue]: transaction is not a ITransaction type

**Review by pstachel — APPROVED** [review]

**Pull request merged**